MekHQ/src/mekhq/gui/dialog/NewPlanetaryEventDialog.java

```diff
@@ -457,97 +409,9 @@ public void actionPerformed(ActionEvent e) {
         gbc.gridx = 3;
         factionCombined = new JLabel();
         pane.add(factionCombined, gbc);
-        
```

Comment:
Question: how would users go about editing planetary info from within MekHQ now?

Reply:
At the moment, they don't, unless they want to brave XML. But I have put back in all the functionality of custom planetary events, so the possibility still exists. My plan is to do a separate planetary system editor in a different PR as this one is quite large already.